content-src/components/ActivityFeed/ActivityFeed.scss

```diff
@@ -9,8 +9,12 @@
 .feed-item {
   display: flex;
   padding-left: $feed-gutter-h + $feed-circle-size;
-  padding-bottom: $feed-gutter-v;
+  padding-right: $feed-gutter-h;
+  padding-bottom: $feed-gutter-v / 2;
+  padding-top: $feed-gutter-v / 2;
   position: relative;
+  border: 1px solid $white;
```

Comment:
The correct thing to do here is use `transparent`, however we're still going to have an issue because there will be a gap appearing in between elements; 

The quick way to fix this is to set the position of the `::before` element to `top: -1px` `bottom: `-1px`, or you could create an inner element that has a border, have the `::before` stuff outside that element.